javascript/code-challenges/tree/tree.js

```diff
@@ -132,25 +194,6 @@ class BinarySearchTree { //create a binary search tree function
     return booleanResult; //return the boolean
   }
 
-  // preOrder (rootNode) { //create preOrder method
-
-  //   let orderedArray = []; //declare empty array for output
-
-  //   function _recursionPre(rootNode) { //create recursive function to traverse tree
-
-  //     if(!rootNode) { //check if root exists, if not return
-  //       return;
-  //     }
-
-  //     orderedArray.push[rootNode.value]; //push the current root value into the output array
-
-  //     _recursionPre(rootNode.left); //re call the function for the left child
-  //     _recursionPre(rootNode.right); //re call the function for the right child
-  //   }
-  //   _recursionPre(rootNode); //invoke the recursive function to start
-    
-  //   return orderedArray; //return the output array
-  // }
 }
 
```

Comment:
Looks like you are off to a wonderful start Simon, once you have time to get back to this and the tests I will regrade for you!

Reply:
All tests are now passing, thanks!